src/StorageSync/StorageSync/help/Invoke-AzStorageSyncFileRecall.md

```diff
@@ -0,0 +1,228 @@
+---
+external help file: Microsoft.Azure.Commands.StorageSync.dll-Help.xml
+Module Name: Az.StorageSync
+online version: https://docs.microsoft.com/en-us/powershell/module/Az.storagesync/invoke-Azstoragesyncfilerecall
+schema: 2.0.0
+---
+
+# Invoke-AzStorageSyncFileRecall
```

Comment:
Rather than using Invoke, we should use a descriptive verb that matches the action undertaken by the services - in this case, that sounds like either 'Reset' or 'Restores'  as in ```Restore-AzStorageSyncFile``` or ```Reset-AzStorageSyncFile```

The list of approved verbs is here: https://docs.microsoft.com/en-us/powershell/developer/cmdlet/approved-verbs-for-windows-powershell-commands

Reply:
These are already approved design. We can accommodate the cmdlet name change in next release

---
In reply to: [255697935](https://github.com/Azure/azure-powershell/pull/8465#discussion_r255697935) [](ancestors = 255697935)